CONTRIBUTORS.md

```diff
@@ -0,0 +1,20 @@
+Abel 'Akronix' Serrano Juste
```

Comment:
These should probably be a bulleted list?  GH render is showing them all on one line